pkg/web/webtest/webtest.go

```diff
@@ -111,9 +112,9 @@ func RequestWithWebContext(req *http.Request, c *contextmodel.ReqContext) *http.
 	return req
 }
 
-func RequestWithSignedInUser(req *http.Request, user *user.SignedInUser) *http.Request {
+func RequestWithSignedInUser(req *http.Request, usr identity.Requester) *http.Request {
```

Comment:
I guess you can leave this as SignedInUser if you are just going to assert it back to SignedInUser after